Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7.3-slim
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV AIRFLOW_HOME=/src/airflow
+
+WORKDIR /src/app
+
+COPY ./requirements ./requirements
+
+RUN pip install -U pip \
+    && set -ex \
+    && apt-get update -yq \
+    && apt-get install -yq --no-install-recommends build-essential \
+    && pip install --no-cache-dir -r ./requirements/base.txt \
+    && apt-get remove --purge --autoremove -yq build-essential \
+    && rm -rf ~/.cache/* \
+    && rm -rf /tmp/*
+
+ARG env
```

Comment:
immutable images should be reusable across environments, so this should be a fixed set of choices (usually two, `dev` and non-`dev`) more than a string with open options; in `search` this is called [`ARG dev`](https://github.com/libero/search/blob/master/docker/Dockerfile#L30), but it's used in a command line option

Reply:
I've updated the docker file to use multistage build so that everything is explicit.